app/models/auth_source.rb

```diff
@@ -25,6 +37,7 @@ class AuthSource < ActiveRecord::Base
   has_many :external_usergroups, :dependent => :destroy
 
   validates :name, :presence => true, :uniqueness => true, :length => { :maximum => 60 }
+  validates_with AuthSourceNameValidator
```

Comment:
Could you do that instead then :smile: moving the validator class to `app/validators` would be appreciated too